infoflow/entropy_kde.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+from infoflow import multidimkde as mdkde
+from math import log
+
+
+def kerneldensityestimation(source, target, timelagx, timelagy, n, bw_coeff):
+    """
+    :param source: source time series, 1-D np.ndarray
+    :param target: target time series, 1-D np.ndarray
+    :param timelagx: time lag in x from present. Integer.
+    :param timelagy: time lag in y from present. Integer.
+    :param n: number of equally spaced points along each dimension where probabilities are to be estimated. Integer.
+    :param bw_coeff: multiplier that adjusts Gaussian bandwidth. Integer.
+    :return: transfer entropy in bits. float.
+    """
+
+    """ All code written here is based on the transfer entropy
```

Comment:
Place this into the docstring above.

"Use protected under GNU" is incorrect.  You can say something like "licensed under the GNU GPL v3 license".